content/api/dashboard_lists/code_snippets/result.api-dashboard-list-get-items.py

```diff
@@ -0,0 +1,85 @@
+{
+    'total': 5,
+    'dashboards': [
+        {
+            'is_shared': False,
+            'author': {
+                'handle': None,
+                'name': None
+            },
+            'url': '/screen/integration/66/aws-dynamodb',
+            'title': 'AWS DynamoDB',
+            'modified': None,
+            'created': None,
+            'is_favorite': True,
+            'is_read_only': True,
+            'type': 'integration_screenboard',
+            'id': 66,
+            'icon': '/static/v/34.254868/images/saas_logos/small/amazon_dynamodb.png'
+        },
+        {
+            'is_shared': False,
+            'author': {
+                'handle': None,
+                'name': None
+            },
+            'url': '/dash/integration/17/postgres---metrics',
+            'title': 'Postgres - Metrics',
+            'modified': None,
+            'created': None,
+            'is_favorite': True,
+            'is_read_only': True,
+            'type': 'integration_timeboard',
+            'id': 17,
+            'icon': '/static/v/34.254868/images/saas_logos/small/postgres.png'
+        },
+        {
+            'is_shared': False,
+            'author': {
+                'handle': 'leo@datadoghq.com',
```

Comment:
Similar to other comments, let's use test1 and test2@datadoghq.com. Never a real email address.